letsencrypt/client/plugins/apache/constants.py

```diff
@@ -0,0 +1,20 @@
+"""Apache plugin constants."""
+import pkg_resources
+
+
+# CLI/IConfig defaults
+DEFAULT_SERVER_ROOT = "/etc/apache2"
+DEFAULT_MOD_SSL_CONF = "/etc/letsencrypt/options-ssl.conf"
+DEFAULT_CTL = "apache2ctl"
+DEFAULT_ENMOD = "a2enmod"
+DEFAULT_INIT_SCRIPT = "/etc/init.d/apache2"
+
+
+MOD_SSL_CONF = pkg_resources.resource_filename(
```

Comment:
I just realized that I forgot to put all `DEFAULT_*` in `CLI_DEFAULTS` (just like I did in core `constants`). I will send a patch in a minute.

Would above resolve the problem, or do you still think this would have to be renamed? IMHO it will not be confusing anymore.